src/pdaaal/engine/PAutomaton.cpp

```diff
@@ -231,19 +241,15 @@ namespace pdaaal {
                             size_t q_new = q_prime[std::make_pair(rule._to, rule._op_label)];
```

Comment:
add an assert that 
`q_prime.find(std::make_pair(rule._to, rule._op_label)) != std::end(q_prime)`

Reply:
Assertion added